packages/codemods/meteor/3/mongo-db-async-methods/.codemodrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://codemod-utils.s3.us-west-1.amazonaws.com/configuration_schema.json",
+  "name": "mongo-db-async-methods",
```

Comment:
should we call this `meteor/v3/mongo-db-async-methods` ? @mohab-sameh

Reply:
yes that's better